qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/completions/QuteCompletionsForExpression.java

```diff
@@ -519,7 +520,7 @@ private void fillCaseOperators(CaseSection caseSection, boolean onlyMulti, Range
 				item.setLabel(operator.getName() + " : Operator");
 				item.setFilterText(operator.getName());
 				item.setKind(CompletionItemKind.Operator);
-				item.setDocumentation(operator.getDocumentation());
+				item.setDocumentation(DocumentationUtils.getDocumentation(operator, true));

```

Comment:
Instead of hard coding with true, you should use the CompletionRequest#canSupportMarkupKind like you do with hover. Compute this flag before the loop.